docs/indyav/onboarding.rst

```diff
@@ -0,0 +1,26 @@
+Getting Started on Indy AV
+==========================
+
+Collaboration
+-------------
+
+Slack
+*****
+Join the slack to keep updated on the developement at `uf-cimar <uf-cimar.slack.com>`_.
```

Comment:
``Join the slack to keep updated on the developement at `uf-cimar <uf-cimar.slack.com>`_.`` -> ``Join the slack to keep updated on the developement at `uf-cimar <https://uf-cimar.slack.com>`_.``